background/index.ts

```diff
@@ -81,3 +81,70 @@ chrome.runtime.onSuspend.addListener(async () => {
 chrome.runtime.onInstalled.addListener(async () => {
   await Promise.all([setClipboardMonitorIsEnabled(true), setupOffscreenDocument(), setupAction()]);
 });
+
+let lastEntries: any = [];
```

Comment:
Avoid global variables in `background/index.ts`. Service workers aren't persistent. `creating` is an edge case for transient concurrency purposes. This also shouldn't be needed if you move the context menu creation logic to `utils/storage.ts:setEntries` as mentioned on Discord.